test/collection/bimap_test.dart

```diff
@@ -40,6 +40,24 @@ void main() {
       expect(map.inverse.isNotEmpty, false);
     });
 
+    test('should support null keys', () {
+      final map = BiMap<String?, int>();
+      map[null] = 5;
+      expect(map.isEmpty, false);
+      expect(map.containsKey(null), true);
+      expect(map.containsValue(5), true);
+      expect(map.keys, contains(null));
```

Comment:
Also `expect(map[null], 5);` :)

Reply:
Might also be worth testing the contents of `map.inverse`

Reply:
Done.